Description: Update lunar.1 man page with added options
 Jidanni reported in 2009:
 .
   There are options, e.g., --utf8, on --help that are not documented on
   the man page. Indeed, also mention --help itself.
 .
 In October 2013, I finally worked on the man page to document the new
 options and changes as found in the Debian version of lunar 2.2-4.
Origin: vendor
Author: Anthony Fok <foka@debian.org>
Bug-Debian: http://bugs.debian.org/514069
Last-Update: 2013-10-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lunar.1
+++ b/lunar.1
@@ -1,29 +1,41 @@
-.TH lunar 1 "28 October 2001"
+.TH LUNAR 1 "4 October 2013" "Lunar version 2.2-4 (Debian)"
+.
 .SH NAME
-lunar (version 2.2) \- a calendar conversion utility
-.SH SYNTAX
-.B lunar
-[
-.B\-h
-] [
-.B\-b
-] [
-.B\-i
-] [
-.B\-l
-]
-.I year month day
-[
-.I hour
-]
+lunar \- a calendar conversion utility
+.
+.SH SYNOPSIS
+.
+.SY lunar
+.OP \-usthg5b
+.
+.SY lunar
+.OP \-usthg5b
+.I year
+.I month
+.I day
+.RI [ hour ]
+.
+.SY lunar
+.OP \-usthg5b
+.B \-i
+.OP \-l
+.I year
+.I month
+.I day
+.RI [ hour ]
+.
+.SY lunar
+.B \-\-help
+.YS
+.
 .SH DESCRIPTION
 The
-.PN lunar
+.I lunar
 program performs date conversion between the Gregorian Solar Calendar (SC)
 and the Chinese Lunar Calendar (LC).
 Given a date in either calendar, the program also
-outputs the corresponding "shengxiao" (animal of the year)
-and "ganzhi" characters.
+outputs the corresponding "\fIsheng\%xiao\fR" (animal of the year)
+and "\fIganzhi\fR" characters.
 The date range currently covered is from about 1900 A.D. to 2049 A.D.
 .PP
 For the sake of convenience, we choose the convention such that the solar 
@@ -39,7 +51,7 @@ and SC 1991.2.16.1 is LC 1991.1.2.1.
 The standard time of the Lunar Calendar is
 Beijing (Hong Kong) Standard Time, not GMT.
 Be sure to adjust appropriately for other time zones
-and "Day-light Saving Time".
+and "Daylight Saving Time".
 .PP
 In the Lunar Calendar,
 a normal year has 12 months, and a leap year (run-nian) has 13 months,
@@ -49,23 +61,63 @@ is called the 6-th leap month.
 A (leap) month is either a short or long one, which has 29 or 30 days
 respectively.
 .PP
-There are 10 gan's and 12 zhi's.
-The ganzhi labeling of the year, month, day and hour of a date
+There are 10 \fIgan\fR's and 12 \fIzhi\fR's.
+The \fIganzhi\fR labeling of the year, month, day and hour of a date
 is a member of the Cartesian product GxGxGxG, where G = {1,2,...,60}.
-For example, "jia-zi" is 1, "yi-chou" is 2, and so on.
-The ganzhi of the j-th leap month is the same as that of the j-th month.
-
+For example, "\fIjia-zi\fR" is 1, "\fIyi-chou\fR" is 2, and so on.
+The \fIganzhi\fR of the j-th leap month is the same as that of the j-th month.
+.
+.SH OPTIONS
+.P
+Since
+.I lunar
+was first packaged for Debian in 2002, the Debian package maintainer
+has made minor additions such as Big5 and UTF-8 support.
+More recently,
+.I lunar
+gained the ability to detect system locale and choose the correct
+character set.
+.P
+.I Lunar
+now also detects and displays the current China Standard Time by default,
+so that the
+.I year
+.I month
+.I day
+.RI [ hour ]
+parameters are now optional for solar-to-lunar calendar conversion.
+.P
 The possible options are
-.IP b
-output ganzhi in special "bitmap" characters.
-.IP h
-output hanzi or Chinese characters encoded in (highest-bit-set) GB code.
-.IP i
+.TP
+.B \-b
+output \fIganzhi\fR in special "bitmap" characters.
+.TP
+\fB\-u\fR, \fB\-\-utf8\fR
+output \fIhanzi\fR or Chinese characters encoded in UTF-8.
+.TP
+.B \-s
+output simplified Chinese (for UTF-8 only).
+.TP
+.B \-t
+output traditional Chinese (for UTF-8 only).
+.TP
+\fB\-g\fR, \fB\-h\fR, \fB\-\-gb\fR
+output \fIhanzi\fR or Chinese characters encoded in (highest-bit-set) GB code.
+.TP
+\fB\-5\fR, \fB\-\-big5\fR
+output \fIhanzi\fR or Chinese characters encoded in Big5.
+.TP
+.B \-i
 convert a lunar date to solar date.
 The default is to convert a solar date to a lunar date.
-.IP l
+.TP
+.B \-l
 indicate the month is a lunar leap month.
-This option is meaningful only when the "-i" option is used.
+This option is meaningful only when the "\-i" option is used.
+.TP
+.B \-\-help
+Print a help message.
+.
 .SH COPYRIGHT
 Copyright (C) 1988, 1989, 1991, 1992, 2001 Fung F. Lee and Ricky Yeung 
 .sp
@@ -88,15 +140,17 @@ version 2.1a.  This program was first re
 GNU GPL on October 28, 2001 as version 2.2. Both versions are identical
 except for the license text.
 
-
 .SH AUTHORS
 Fung F. Lee and Ricky Yeung
+.
 .SH ACKNOWLEDGEMENT
 The special "bitmap" file "lunar.bitmap" was contributed by Weimin Liu.
 
 Special thanks to Hwei Chen Ti who extended the tables from 2001 to 2049.
+.
 .SH BUGS
 Bug reports and comments should be sent to lee@umunhum.stanford.edu.
+.
 .SH DISCLAIMER
 This software has no connection with our employers.
 
